package.json

```diff
@@ -91,6 +91,8 @@
     "test:watch": "jest --watch --verbose",
     "lint": "eslint 'src/**/*.{ts,tsx}'",
     "lint:fix": "eslint --fix 'src/**/*.{ts,tsx}'",
+    "prettier": "prettier --config .prettierrc.json --check 'src/**/*.{ts,tsx}'",
+    "prettier:fix": "prettier --config .prettierrc.json --write 'src/**/*.{ts,tsx}'",
```

Comment:
I would say since this hasn't been run, just remove this completely and add it in when you run prettier on the files.

Logical changes for posterity - in the future, after this falls out of context, our intent will make much more sense if we group these prettier changes together. It's a bit more work and tedium now, but it pays off a lot for future Monitoring Team.

Reply:
It has been run on all files. I kept the prettier changes for the files I changed. I backed out prettier changes to files I didn't change.

Reply:
I made a separate issue for prettier.